src/utils/index.js

```diff
@@ -7,12 +7,24 @@ dotenv.config();
  * @return {string} generated token that is a string data type.
  */
 export const emailVerifyToken = (id) => new Promise((resolve) => {
-  const token = jwt.sign({ id }, process.env.SECRET_KEY_EMAIL_VERIFY_TOKEN, { expiresIn: '24h' });
+  const token = jwt.sign({ id }, process.env.EMAIL_VERIFY_TOKEN_SECRET_KEY, { expiresIn: '24h' });
```

Comment:
This is conflicting with what you have in your .env.example
correct `EMAIL_VERIFY_TOKEN_SECRET_KEY` to `SECRET_KEY_EMAIL_VERIFY_TOKEN` in
`firestar-backend/src/utils/index.js`

Reply:
I have two tokens. token for email generation and token for accessing protected route. I used different secret keys to generate and verify token